src/main/java/duke/commands/FindCommand.java

```diff
@@ -0,0 +1,34 @@
+package duke.commands;
+import java.util.ArrayList;
```

Comment:
Maybe you can leave a line after package statement for consistency with your other files.